### DISCUSSION TITLE
The workbench failed to connect to the server (Error: undefined)

### DISCUSSION THREAD

**Discussion by evanvarian11** [coder/code-server#4750] [Q&A]
After successfully deploy "code-server vscode" on my server and login success i see this undefined error

**An unexpected error occurred that requires a reload of this page.
The workbench failed to connect to the server (Error: undefined)**

page show up and it can run on browser but that's pop up error prevent me to do anything since it will reload when i closed it

I check console.log too from chrome and show some error like this

```
errors.ts:28 Uncaught Error: No file system provider found for resource 'vscode-remote:/var/lib/code-server/User/settings.json'
errors.ts:28 Uncaught Error: No file system provider found for resource 'vscode-remote:/var/lib/code-server/User/tasks.json'
```

my nginx proxy pass conf
```
location /vscode/ {
       #include proxy_params;
       proxy_pass  http://127.0.0.1:7171/;
       # WebSocket support
       proxy_http_version 1.1;
       proxy_set_header Upgrade $http_upgrade;
       proxy_set_header Connection "upgrade";
}
```

never find this similar issue so i came to ask it

code-server version 4.0.1
web server nginx
os ubuntu 20

**Comment by jsjoeio** [comment]
Hmmm...sounds like it could be an issue with your nginx conf? I'm not too sure though. You might have to dig through Discussions and look at other confs. 

How did you install code-server and how are you running it?
